packages/vue/src/components/sign-in.vue

```diff
@@ -134,27 +129,24 @@ const onForgotPasswordClicked = (): void => {
                 : signInButtonText
             }}
           </amplify-button>
-          <amplify-button
-            class="amplify-field-group__control"
-            data-fullwidth="true"
-            data-size="small"
-            data-variation="link"
-            style="font-weight: normal"
-            type="button"
-            @click.prevent="onForgotPasswordClicked"
-          >
-            {{ forgotYourPasswordLink }}
-          </amplify-button>
         </base-wrapper>
-        <base-footer>
-          <slot
-            name="footer"
-            :onSignInSubmit="onSignInSubmit"
-            :onForgotPasswordClicked="onForgotPasswordClicked"
-          >
-          </slot>
-        </base-footer>
       </base-form>
     </base-wrapper>
+
+    <base-footer>
+      <slot name="footer">
+        <amplify-button
+          @click="onForgotPasswordClicked"
```

Comment:
@ErikCH  Is there a reason this file isn't using `toForgotPassword` and instead calling `send(...)`?

Reply:
I just haven't gone back through all the internals of the Vue code to use the new useAuthenticator. I figure post GA I can do that.

Reply:
Cool, we can leave this then – it only really matters for the documented examples anyway 👍